test/plugin/test_in_kubernetes_metrics.rb

```diff
@@ -35,6 +35,9 @@ class KubernetesMetricsInputTest < Test::Unit::TestCase
   ).freeze
 
   setup do
+    stub_k8s_requests
+
+    return unless @@hash_map_test.empty?
```

Comment:
Test setup was called on every testcase execution and took too much time executing unit test